libraries/AP_HAL_ChibiOS/hwdef/CubeOrange/defaults.parm

```diff
@@ -1,11 +1,19 @@
 # setup correct defaults for battery monitoring for cube power brick
 BATT2_CURR_PIN 4
 BATT2_VOLT_PIN 13
+BATT_MONITOR 4
 BATT_AMP_PERVLT 39.877
 BATT_VOLT_MULT 12.02
+BATT2_MONITOR 4
```

Comment:
this will give arming failure after this PR:
https://github.com/ArduPilot/ardupilot/pull/21012